CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+List of end-devices can now be sorted by `last_seen_at` field. Devices with the field value `NULL` will be showned last in the ascending order.
```

Comment:
Don't forget to start with `-`. Also users do not really conceptualize what `NULL` means here, so consider formulating this as 'unseen devices will be shown last'.